src/nodedef.cpp

```diff
@@ -890,13 +890,13 @@ void CNodeDefManager::updateTextures(IGameDef *gamedef,
 				for (u32 i = 0; i < 6; i++)
 					tiledef[i].name += std::string("^[noalpha");
 			}
-			if (f->waving == 1)
+			if (f->waving >= 1)
 				material_type = TILE_MATERIAL_WAVING_LEAVES;
 			break;
 		case NDT_PLANTLIKE:
 			f->solidness = 0;
 			f->backface_culling = false;
-			if (f->waving == 1)
```

Comment:
this avoids plants waving like leaves if someone accidentally chooses to do so, but still makes them wave, instead of not waving at all with `waving = 2`.